.github/CHANGELOG.md

```diff
@@ -71,6 +25,39 @@
 
   See the `default.qubit.autograd` documentation for more details.
 
+<h4>New algorithms and templates</h4>
+
+* Added built-in QAOA functionality via the new `qml.qaoa` module.
+  [(#712)](https://github.com/PennyLaneAI/pennylane/pull/712)
+  [(#718)](https://github.com/PennyLaneAI/pennylane/pull/718)
+  [(#741)](https://github.com/PennyLaneAI/pennylane/pull/741)
+  [(#720)](https://github.com/PennyLaneAI/pennylane/pull/720)
+
+  ```pycon
+  >>> graph = nx.Graph([(0, 1), (1, 2)])
+  >>> cost_h, mixer_h = qml.qaoa.maxcut(graph)
+  ```
```

Comment:
I think an extended example should go here. I was having a look in the docstrings, and there is no 'complete' QAOA example anywhere. A good place to put it would be the QAOA module page: https://pennylane.readthedocs.io/en/latest/code/qml_qaoa.html

Reply:
Perhaps @Lucaman99 can help with this?

Reply:
@co9olguy For sure, I'll add this today

Reply:
I've added in your example @Lucaman99!

Reply:
Awesome, thank you @josh146!